dns/dnscrypt-proxy/src/opnsense/scripts/OPNsense/Dnscryptproxy/dnsbl.sh

```diff
@@ -276,14 +276,14 @@ for CAT in $(echo ${DNSBL} | tr ',' ' '); do
 	hup)
 		hphosts-pup
 		;;
-	ht)
-		hbbtv
+	st)
+		smarttv
 		;;
 	nc)
 		nocoin
 		;;
 	rw)
```

Comment:
@mimugmail maybe we should delete all rwtracker code if it is removed from the selection?